r4/resource-apis/account/Config.toml

```diff
@@ -0,0 +1,4 @@
+# sourceSystem = "http://localhost:9595"
```

Comment:
Do we need this file? We don't use sourceSystem anymore and info logs will anyways be printed.

Reply:
Removed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.templates/commit/10d99e6600a555498933a04733525050c16746f1